private/bufpkg/bufmodule/bufmoduleapi/uploader.go

```diff
@@ -449,7 +457,7 @@ func getV1Beta1ProtoUploadRequestContent(
 		return nil, syserror.New("expected local Module in getProtoLegacyFederationUploadRequestContent")
 	}
 	if module.ModuleFullName() == nil {
-		return nil, syserror.Newf("expected module name for local module %q", module.OpaqueID())
+		return nil, syserror.Newf("expected module name for local module %q", module.Description())
```

Comment:
Description vs opaqueID does not really matter for syserrors, but I changed it just in case.